horace_core/utilities/@sqw/symmetrise_sqw.m

```diff
@@ -122,7 +124,8 @@
 
 %Note that we allow the inclusion of an offset from the origin of the
 %reflection plane. This is specified in rlu.
-vec3=uconv\(v3'-header.uoffset(1:3));
+%vec3=uconv\(v3'-header.uoffset(1:3));
```

Comment:
is the header offset no-longer used in this calculation and the vec1 & vec2 calculations above?

Reply:
I believe it does not but I am not entirely sure. This is why it is left as it is -- symmetrisation work left unfinished.